content/community-targets/ATOM_Lite_ESP32.md

```diff
@@ -0,0 +1,61 @@
+# M5Stack ATOM Lite ESP32
```

Comment:
The image for this device is available from the official distribution since November 2020.
And there is nothing about it in the community repo, so it can be misleading.

Suggest that you grab the image and description here of the device and rework the ESP32 one, by adding it there.